cli/cmd/start/start.go

```diff
@@ -182,23 +173,27 @@ func StartCmd(ch *cmdutil.Helper) *cobra.Command {
 	}
 
 	startCmd.Flags().SortFlags = false
+	startCmd.Flags().StringSliceVarP(&envVars, "env", "e", []string{}, "Set environment variables")
+	startCmd.Flags().StringVar(&environment, "environment", "dev", `Environment name`)
+	startCmd.Flags().BoolVar(&reset, "reset", false, "Clear and re-ingest source data")
 	startCmd.Flags().BoolVar(&noOpen, "no-open", false, "Do not open browser")
+	startCmd.Flags().BoolVar(&verbose, "verbose", false, "Sets the log level to debug")
+	startCmd.Flags().BoolVar(&readonly, "readonly", false, "Show only dashboards in UI")
 	startCmd.Flags().IntVar(&httpPort, "port", 9009, "Port for HTTP")
 	startCmd.Flags().IntVar(&grpcPort, "port-grpc", 49009, "Port for gRPC (internal)")
-	startCmd.Flags().BoolVar(&readonly, "readonly", false, "Show only dashboards in UI")
 	startCmd.Flags().BoolVar(&noUI, "no-ui", false, "Serve only the backend")
-	startCmd.Flags().BoolVar(&verbose, "verbose", false, "Sets the log level to debug")
 	startCmd.Flags().BoolVar(&debug, "debug", false, "Collect additional debug info")
-	startCmd.Flags().BoolVar(&reset, "reset", false, "Clear and re-ingest source data")
 	startCmd.Flags().StringVar(&logFormat, "log-format", "console", "Log format (options: \"console\", \"json\")")
 	startCmd.Flags().StringVar(&tlsCertPath, "tls-cert", "", "Path to TLS certificate")
 	startCmd.Flags().StringVar(&tlsKeyPath, "tls-key", "", "Path to TLS key file")
-
-	// --env was previously used for variables, but is now used to set the environment name. We maintain backwards compatibility by keeping --env as a slice var, and setting any value containing an equals sign as a variable.
-	startCmd.Flags().StringSliceVarP(&env, "env", "e", []string{}, `Environment name (default "dev")`)
-	startCmd.Flags().StringSliceVarP(&vars, "var", "v", []string{}, "Set project variables")
 	startCmd.Flags().StringSliceVarP(&allowedOrigins, "allowed-origins", "", []string{}, "Override allowed origins for CORS")
 
+	// Deprecated support for "--env": replaced by "--env".
```

Comment:
// Deprecated support for "--var": replaced by "--env" ?

Reply:
Thanks!